AdyenActions/Actions/VoucherAction.swift

```diff
@@ -127,15 +135,6 @@ public class GenericVoucherAction: Decodable,
     /// Merchant Name.
     public let merchantName: String
 
-    /// The instruction url.
-    @available(*, deprecated, message: "Please use `instructionsURL` instead.")
-    public var instructionsUrl: String {
-        instructionsURL.absoluteString
-    }
-    
-    /// The instruction `URL` object.
-    public let instructionsURL: URL
```

Comment:
This is public class and above is a breaking change 🤔
Probably ok to do, but still - slippery slope..

Reply:
as discussed, this can pass for now.